regression-test/suites/join/ddl/test_bucket_shuffle_join.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE test_bucket_shuffle_join (
+  `id` int NOT NULL COMMENT "",
+  `rectime` datetime NOT NULL COMMENT ""
+)
+UNIQUE KEY(`id`,`rectime`)
+COMMENT "olap"
+PARTITION BY RANGE(rectime) ()
+DISTRIBUTED BY HASH(id)
+PROPERTIES
+(
+ "replication_num" = "1",
```

Comment:
Better not using `dynamic_partition`, because it will change as time pass by.

Reply:
no need matter the thing, we only use the `history partition` in the case.